Makefile

```diff
@@ -20,6 +20,10 @@ show-coverage:
 e2e-tests:
 	behave --tags=-skip
 
+e2e-tests-fast: export BEHAVE_LOCAL_ENV=TRUE
+e2e-tests-fast:
+	behave --tags=-skip --no-capture
```

Comment:
Can't this be

```suggestion
e2e-tests-fast:
	BEHAVE_LOCAL_ENV=TRUE behave --tags=-skip --no-capture
```

?

Reply:
I tried this but it doesn't work. I am not an expert of Makefile so I am happy with change if you found a fix.

<img width="539" alt="image" src="https://github.com/kedro-org/kedro/assets/18221871/a2a998b4-8321-4c67-a594-8cc784872e7a">